webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/resource/visualizer_ui.js

```diff
@@ -344,7 +344,11 @@ var VisualizerUI = (function($, window, undefined) {
                     }
 
                     comment += '</div>';
-                    comment += ('<div class="comment_text">"' + Util.escapeHTML(spanText) + '"</div>');
+                    if(hoverSpantext)
+                    	comment += ('<div class="comment_text">' + Util.escapeHTML(hoverSpantext) + '</div>');
+                    else
+                    	if(spanText)
+                    		comment += ('<div class="comment_text">"' + Util.escapeHTML(spanText) + '"</div>');
```

Comment:
Why do we need `hoverSpantext` and `spanText`?

Reply:
hoverSpantext is not a good name, I changed it to hoverText (see a9ab7a99bedd38f85e272725b0254532d815f1b1)

Reply:
We should keep both in order to have a default behavior if hoverText is not provided. Also, spanText is extracted by brat and not provided by the WebAnno API whereas hoverText is provided by the WebAnno API even if it contains the spanText